src/fragments/lib/logging/common/configure_user/configure_user.mdx

```diff
@@ -0,0 +1,34 @@
+Each Amplify authenticated user can be configure with their own unique logging configuration.  The Amplify user's AWS Cognito User Pool Sub is used as the user identifier. 
+
+## Getting the Cognito User Pool Sub
+You can use the Amplify Auth category to get the Cognito User Pool Sub.
+
+import iosCurrentUser from '/src/fragments/lib/logging/ios/configure_user/auth_current_user.mdx';
+import androidCurrentUser from '/src/fragments/lib/logging/android/configure_user/auth_current_user.mdx';
+
+<Fragments
+    fragments={{
+        ios: iosCurrentUser,
+        android: androidCurrentUser
+    }}/>
+
+## Configuring Allow List
+
+Below is an example of setting different a default log level and category level for an authenticated user with AWS Cognito User Pool sub `xyz-123`.
```

Comment:
Small tweaks:

Below is an example of setting different default and category log levels for an authenticated user.

Reply:
Text updated.